tcptype.go

```diff
@@ -0,0 +1,14 @@
+package ice
+
+// TCPType is the type of ICE TCP candidate as described in
+// ttps://tools.ietf.org/html/rfc6544#section-4.5
+type TCPType string
+
+const (
+	// TCPTypeActive is active TCP candidate, which initiates TCP connections.
+	TCPTypeActive TCPType = "active"
```

Comment:
I usually do `iota + 1`, the 'no default value' is nice for debugging!

Reply:
Updated to `int`s